homeassistant/components/kaiterra/api_data.py

```diff
@@ -87,10 +87,11 @@ async def async_update(self) -> None:
                         main_pollutant = POLLUTANTS.get(sensor_name)
 
                 level = None
-                for j in range(1, len(self._scale)):
-                    if aqi <= self._scale[j]:
-                        level = self._level[j - 1]
-                        break
+                if aqi:
```

Comment:
What happens now if aqi is 0?
If we specifically want to guard against aqi being `None`:
```suggestion
                if aqi is not None:
```